scripts/lightweight_curator.py

```diff
@@ -94,37 +95,49 @@ def get_first_item(a_dict={}):
     first_value = next(value_iterator)
     return first_value
 
+def index_smaller_then_max_allowed_size(index, limit, indices_size_counter, indices_to_delete):
```

Comment:
If this returns a list then imo it shou be plural
I know it's very confusing where to put which term but you get the hang of it after a while
```suggestion
def indices_smaller_then_max_allowed_size(index, limit, indices_size_counter, indices_to_delete):
```